templates/federated_reporting/distributed_cleanup.py

```diff
@@ -253,6 +254,12 @@ def main():
             sys.exit(1)
 
     fr_distributed_cleanup_password = read_secret(DISTRIBUTED_CLEANUP_SECRET_PATH)
+    if args.setup_feeder:
+        email = input("Enter email for fr_distributed_cleanup accounts: ")
```

Comment:
This should handle non-interactive mode or it should fail properly (and the help output should mention it too).

Reply:
will check.